apps/system/safe_mode.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
```

Comment:
nit: We use `<!DOCTYPE html>` mostly everywhere in gaia, let's keep it consistent.